utilities/implementation-kit/data-migration/Tenant _CreationOrUpdation/create_tenants_json.py

```
@@ -0,0 +1,142 @@
+import openpyxl
+import json
+
+json_file = "tenants.json"
+# Opening JSON file
+f = open(json_file)
```

Ruff

Use a context manager for opening files

(SIM115)